packages/nimble-components/src/icon-base/tests/icon-metadata.ts

```diff
@@ -16,6 +16,9 @@ export const iconMetadata: {
     IconArrowDown: {
         tags: ['sort', 'arrow', 'down', 'descending']
     },
+    IconArrowDownBox: {
```

Comment:
Think it would be more consistent with other icons to be arrow-down-rectangle and arrow-up-rectangle, i.e. we have:
- nimble-icon-rectangle-check-lines
- nimble-icon-true-false-rectangle
- nimble-icon-calendar-rectangle

We don't have any other "box" (we do have checkbox: nimble-icon-indeterminant-checkbox)

Reply:
Makes sense to me! @NIbokeefe FYI I'm going to slightly change the names of these icons to match Milan's proposal unless you object.
arrow-down-box -> arrow-down-rectangle
arrow-up-box -> arrow-up-rectangle

Reply:
Do these icons have the same purpose as the Font Awesome `inbox-in` and `inbox-out` icons?

<img width="285" alt="Screenshot 2024-09-05 at 10 50 52 AM" src="https://github.com/user-attachments/assets/3732b969-9e94-40f3-a67f-35a0b31e6704">

They share a similar aesthetic, but perhaps they are different? If it's a close call, I'd still go with `arrow-up/down-rectangle`

Reply:
I agree, similar aesthetic, but I think different meaning. I see the "inbox" as being specifically email-related while these are more about packages / software.